cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/common/Constants.java

```diff
@@ -74,6 +74,21 @@ private Constants() {
     public static final String AGG_GROUPS = "aggregator.groups";
     public static final String JOIN_KEYS = "joiner.keys";
     public static final String DRAFT_COUNT = "draft.count";
+
+    public static final class Connection {
+      public static final String CONNECTION_COUNT = "connection.count";
```

Comment:
nit: should the global metrics be connections (e.g. connections.count) to differentiate them from the per connection type below. 

(Similar to how we do records above)

Reply:
I made all of them connections